src/messaging/ReliableMessageMgr.cpp

```diff
@@ -511,6 +546,22 @@ void ReliableMessageMgr::CalculateNextRetransTime(RetransTableEntry & entry)
         peerIsActive = sessionHandle->AsUnauthenticatedSession()->IsPeerActive();
     }
 
+    // For initial send the packet has already been submitted to transport layer successfully.
```

Comment:
How do people feel about the location of this and how re-transmission has technically not taken place yet